server/echo_service_test.go

```diff
@@ -450,3 +373,12 @@ func TestPagedExpand(t *testing.T) {
 		}
 	}
 }
+
+func TestWait(t *testing.T) {
+	waiter := &mockWaiter{called: false}
+	server := &echoServerImpl{waiter: waiter}
+	server.Wait(context.Background(), nil)
+	if !waiter.called {
```

Comment:
I'm not exactly following. Can you give an example?

Reply:
Since `WaitRequest` has the time it should wait until, I thought it might be useful to record, something like

```go
type mockWaiter struct {
  req *pb.WaitRequest
}

func (w *mockWaiter) Wait(req *pb.WaitRequest) {w.req = req}
```

Reply:
Ah yeah. Done!